File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int x = scanner.nextInt();
        int y = scanner.nextInt();
        int z = scanner.nextInt();
        int p = 0;

        if(x == y && y == z) {
            p = 10000 + 1000 * x;
        } 
        else if (x != y && y != z && x != z) {
            int max = Math.max(Math.max(x, y), z);
            p = 100 * max;
        }
        else if (x == y || x == z) {
            p = 1000 + 100 * x;
        }
        else {
            p = 1000 + 100 * y;
        }

        System.out.println(p);
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
A, B, C = map(int, input().split())

if A == B == C :
    print(10000 + 1000 * A)
elif A != B and B != C and A != C:
    L = max(A, B, C)
    print(100 * L)
elif A == B or A == C :
    print(1000 + 100 * A)
else :
    print(1000 + 100 * B)